### DISCUSSION TITLE
Ideas para e-commerce

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#2] [Ideas]
Creo esta discusión para ir aportando ideas para nuestro nuevo proyecto.

Recuerden que debemos tener completa la documentación para el viernes 14/04.

**Comment by AhumadaCeleste** [comment]
Como idea para el proyecto, se me ocurre una  heladería,  dado a que con pocas entidades podemos crear una estructura básica y necesaria para alcanzar los objetivos. 

El usuario se registra, selecciona el producto, sabores, etc, opta por el servicio de envió a domicilio o retiro en tienda, se genera el pago y se emite la factura. 

En cuanto al abono, quizás pudiéramos transformar un un sistema de puntos para cambiar por algún beneficio para el cliente.

Imagino la posibilidad de sumarle complejidad, una vez cumplimentado lo solicitado, incorporando alguna api en la cual,  al optar por el  envió a domicilio, calcule distancia y en base a ello el costo del  servicio.

**Comment by AhumadaCeleste** [comment]
Propongo realizar un  e-comerce orientado a la venta de productos del tipo cosmética.

El el podríamos sumar diferentes tipos de servicio de entrega por correo o envió mediante aplicación y vender algún tipo de consulta video llamada con especialista en dermatología, cosmetología etc. 

También generar al cliente algún beneficio del tipo descuento, en caso de reponer el envase por reposición, como fin de promover el reciclado y disminuir el costo del empaquetado.

Podríamos crear, combos de productos con descuentos especiales por determinado plazo de tiempo,  beneficio por cumpleaños en el cual se notifique al beneficiario el mismo o vender alguna Gift card de obsequio. 
 
Podríamos tener una pagina con rutina de cuidado en el cual agreguemos los vínculos para la venta de esos determinados productos.

Organizar las tablas en tipo e producto como: Cremas, Cuidado personal, Cabello etc. 

Imagino otra pagina tipo blog donde los usuarios puedan compartir la experiencia con los productos.


Dejo mi propuesta a consideración del grupo.

**Comment by rpgrca** [comment]
La de los helados me parece demasiado simple aunque la lógica cerraría si le agregamos delivery y tal vez hasta un abono mensual. La de la cosmética me parece más completo aunque evitaría mencionar lo de la teleconsulta por lo complicado que podría ser (porque habría que agregar tablas para los turnos, para los médicos y sus especialidades, etc). Todo lo que mencionemos hay que tenerlo listo para el sprint 1 así que hay que tener cuidado con no proponer demasiado. Estuve pensando desde la mañana y creo que me quedo con lo de la heladería porque es algo más simple a nivel lógica porque solo se debería filtrar por tipo (crema o agua por ejemplo) y el sabor mientras que el de la cosmética me parece que se puede volver tan complicado como la ropa deportiva.

**Comment by NataliaAlvarezIspc** [comment]
Buenas! paso a dar mi opinión.

Con respecto a la de la Heladería, me parece bastanten simple, Tenemos que tener los gustos y se vendería cuarto, medio kilo y un kilo, no veo como implementar lo del abono, pero se podría hacer que cada tanto importe de compra x mes se haga un descuento para el próximo kilo (que sería algo parecido a lo que comenta Celeste con respecto a los puntos). También se pueden crear combos ej: cada 2 kg., un cuarto de regalo.

Con respecto a la idea de Productos Cosmética, me gusta la idea aunque la simplificaría. Le quitaría la consulta telefónica y la rutina de cuidado, creo que complicaría la implementación.

De todas formas me sumo a lo que vote la mayoría.

**Comment by claubott** [comment]
Hola! Me gustan las dos ideas de Celeste. La de cosmética natural me parece más interesante porque es un escenario real (no estaríamos inventando un negocio, sino creando un ecommerce para un negocio de verdad), pero estoy de acuerdo con no incluir lo de las videoconsultas.
La de la heladería también me gusta y parece más fácil. Y se me ocurre que para incluir algún abono se pueden ofrecer membresías, como los clubes de vino que te mandan unas botellas todos los meses. Podría haber varios niveles que incluyan más o menos beneficios según el valor del abono. Se podría incluir un sistema de puntos y descuentos (como ya mencionaron) o simplemente enviar algún producto semanalmente o mensualmente (sabores nuevos, algún helado gourmet, el postre para los domingos, etc.). 
Una membresía se puede aplicar a casi cualquier negocio. Y se puede mantener lo más simple posible. Supongo que lo importante es incorporar un pago recurrente, que se diferencie de cómo se manejan las compras de productos y servicios.

**Comment by Brian-Almada** [comment]
Por mi parte, me sumo a la idea de la heladería. Me parece que tiene la simpleza que necesitamos para los tiempos acotados de entrega del trabajo. Además al trabajar en una pastelería se me hace más familiar ese tipo de negocio que la de los cosméticos. 
Creo que estaría bueno para la heladería que la gente se pueda hacer "socia" con una subscripción como la que te da Grido cuando te hacer parte del ClubGrido. Así la persona tendría ofertas especiales, rebajas o combos, o el envío gratuito, etc.

**Comment by NataliaAlvarezIspc** [comment]
Decidido que trabajaremos sobre el proyecto de la heladería. Cierro la discusión.
